signal_test.go

```diff
@@ -0,0 +1,490 @@
+package dbus_test
```

Comment:
Ok, I sidestepped the issue by removing the use of introspection. It's not mandatory for dbus server and it wasn't used at all in these tests.